docs/sphinx/formats/perkinelmer-vectra-qptiff.rst

```diff
@@ -35,7 +35,7 @@ We would like to have:
 
 Pixels: |Very good|
 
-Metadata: |Very good|
+Metadata: |Outstanding|
```

Comment:
Is this edit to an unrelated format intentional?

Reply:
Yes, as mentioned in the PR description, the rating was auto-edited after running the `metadata-ratings` target. I included it as part of this PR. Can push it away but it would be part of the doc autogeneration step performed prior to each release anyways.

Reply:
I think you may as well leave it here